$OpenBSD: patch-test_rubygems_test_gem_dependency_installer_rb,v 1.1 2010/09/23 21:58:50 jeremy Exp $

Make the regression test suite deal with a default of format_executable=>true.

--- test/rubygems/test_gem_dependency_installer.rb.orig	Wed Sep  8 15:17:52 2010
+++ test/rubygems/test_gem_dependency_installer.rb	Wed Sep  8 14:49:42 2010
@@ -265,7 +265,7 @@ class TestGemDependencyInstaller < RubyGemTestCase
     inst = nil
 
     Dir.chdir @tempdir do
-      inst = Gem::DependencyInstaller.new :env_shebang => true, :wrappers => true
+      inst = Gem::DependencyInstaller.new :env_shebang => true, :wrappers => true, :format_executable => false
       inst.install 'a'
     end
 
@@ -524,7 +524,7 @@ class TestGemDependencyInstaller < RubyGemTestCase
     def test_install_no_wrappers
       @fetcher.data['http://gems.example.com/gems/a-1.gem'] = read_binary(@a1_gem)
 
-      inst = Gem::DependencyInstaller.new :wrappers => false
+      inst = Gem::DependencyInstaller.new :wrappers => false, :format_executable => false
       inst.install 'a'
 
       refute_match(%r|This file was generated by RubyGems.|,
